sql-plugin/src/main/scala/com/nvidia/spark/rapids/literals.scala

```diff
@@ -176,6 +176,8 @@ object GpuScalar extends Arm with Logging {
         val colType = resolveElementType(elementType)
         val rows = seq.map(convertElementTo(_, elementType))
         ColumnVector.fromStructs(colType, rows.asInstanceOf[Seq[HostColumnVector.StructData]]: _*)
+      case NullType => // Byte is used for NullType
+        ColumnVector.fromBoxedBytes(seq.asInstanceOf[Seq[JByte]]: _*)
```

Comment:
Why we are using Byte for NullType, pls refer to https://github.com/NVIDIA/spark-rapids/blob/branch-21.06/sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuColumnVector.java#L456

Reply:
So here a ColumnVector filled with nulls is expected ? If yes, you can try
`GpuColumnVector.columnVectorFromNull(seq.size, NullType)`, then you do not need the comment here.

Reply:
Yes, cudf does not support a null type so we just use a byte because it is the smallest available. and @firestarman is correct please use `GpuColumnVector.columnVectorFromNull(seq.length, NullType)`

Reply:
Done

Reply:
@revans2 Seems seq.size equals seq.length

Reply:
Yes, it is super minor. Scala prefers `length` over `size`. Java has `length` for arrays and `size` for everything else. In Scala both work but idiomatic Scala prefers length, so it is consistent everywhere.